components/moderation-panel/ModEditSubmissionForm.vue

```diff
@@ -671,7 +671,7 @@ async function submitCompletedForm(e: Event) {
         return
     }
 
-    if (!hasEnglishName) {
+    if (!hasEnglishName && !currentExistingHealthcareProfessionals.value.length) {
```

Comment:
I think we can optimize this a bit AND make it more readable. Let's skip validations like `hasEnglishName` if there are no active healthcare professional changes.